guides/theme-quality-assurance.md

```diff
@@ -0,0 +1,295 @@
+  * IE 11+
+  * Edge 16+
+  * Opera 53+
+* Mac
+  * Safari 10+
+  * Chrome C66+
+  * Firefox 60+
+  * Opera 53+
+
+### Tablet
+
+* Android
+  * Current Galaxy
+  * Marketshare Winner
+* iOS
+  * Current iPad Pro
+  * Current iPad
+
+### Mobile
+
+* Android
+  * Current Galaxy
+  * Current Pixel
+  * Marketshare Winner
+* iOS
+  * Current Large
+  * Current Small
+  * Marketshare Winner
+
+---
+
+## Theme Sections
+
+### Homepage/Overall
+
+* Homepage is responsive and behaves normally and beautifully at phone, tablet and desktop widths. Measure at varying heights and widths. View window should not obscure any tap targets.
+* Homepage products link properly to their products
+* Homepage navigation is present and consistent throughout the site
+* Client’s brand colors are present and consistent throughout the site
+* Typography is branded and in client’s choice of brand font
+* Social, Meta tags and SEO tags are present and well formed
+* Pixels are present and firing (page view, lead if mail capture)
+* Google Analytics is present and tracking
+* Tap/Click targets are large enough to be clicked properly
+* Body copy is engaging, present and branded to the client
+* Animations are present, but lightweight and functional (enhance, don’t rely)
+* Where Javascript and animation has been deemed necessary, an elegant & workable fallback is offered to support legacy browsers.
+
+### Navigation
+
+* Navigation is responsive and allows for site traversal on mobile and desktop
+* Navigation is clear and links are well formed
+* Navigation leads to the expected locations
+* Navigation hover angles follow Amazon hover convention if mega-menu
+* Tap/Click targets are large enough to be clicked properly
+
+### Footer
+
+* Footer behaves properly on mobile & desktop
+* Social Links are present, branded and functioning
+* Footer links are well formed
+* Footer has copyright present and some form of trust enhancer
+* Tap/Click targets are large enough to be clicked properly
+* Footer links include refs to Privacy Policy and T&C
+
+### Collection
+
+* Collection page is responsive and behaves normally and beautifully at phone, tablet and desktop widths. Measure at varying heights and widths. View window should not obscure any tap targets.
+* Filtering capabilities work across devices and are friendly to interact with
+* Breadcrumbs (if selected are present and functioning)
+* Social, Meta tags and SEO tags are present and well formed
+* Pixels are present and firing (category view, page view)
+* Products can be sorted
+* Products respect sorting in admin
+* Products link to correct products
+* Tap/Click targets are large enough to be clicked properly
+* When a new product is created and tagged with a collection, it shows up in that collection properly
+
+### Product
+
+* Product page is responsive and  normally and beautifully at phone, tablet and desktop widths. Measure at varying heights and widths. View window should not obscure any tap targets.
```

Comment:
I'm unsure of the correct way to write this but in its current state, this sentence structure is a bit confusing: "...responsive and normally and beautifully...". Maybe it's missing a word and/or has an extra word in it?